tox.ini

```
@@ -29,13 +32,11 @@ setenv =
 deps =
     -e.[test]
     psycopg2-binary
-    django111: Django>=1.11,<2.0
-    django111: djangorestframework<3.12
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
     django22: Django>=2.2,<3.0
-    django30: Django>=3.0a1,<3.1
+    django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django32: Django>=3.2,<4.0
+    django40: Django>=4.0,<5.0
```

We probably want to exclude the future 4.1 and 4.2 releases here.
```suggestion
    django40: Django>=4.0,<4.1
```